src/main/java/core/basesyntax/FigureSupplier.java

```diff
@@ -0,0 +1,36 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class FigureSupplier {
+    private ColorSupplier colorSupplier = new ColorSupplier();
+
+    public Figure getRandomFigure() {
+        int figureIndex = new Random().nextInt(4);
```

Comment:
```suggestion
    public Figure getRandomFigure() {
        int figureIndex = ranodm.nextInt(4);
```
make random global as well